public/walkthroughs/1a/walkthrough.adoc

```diff
@@ -40,6 +40,25 @@ This walkthrough describes how  to create such an integration, using OpenShift,
 
 // end::master-1[]
 
+
+[type=walkthroughResource]
```

Comment:
Why not use `serviceName=fuse` here?

Reply:
How about we do a follow on task of https://github.com/integr8ly/tutorial-web-app/issues/307
I think there are a number of items to discuss